src/main/java/seedu/todo/commons/util/DateUtil.java

```diff
@@ -46,21 +54,25 @@ public static String formatDay(LocalDateTime dateTime) {
         LocalDate date = dateTime.toLocalDate();
         long daysDifference = LocalDate.now().until(date, ChronoUnit.DAYS);
 
-        String fromNow = "later";
-        String tillNow = "ago";
-
         // Consider today's date.
-        if (date.isEqual(LocalDate.now()))
-            return "Today";
+        if (date.isEqual(LocalDate.now())) {
+            return TODAY + SPACE + date.getDayOfMonth() + SPACE + date.getMonth();
+        }
+        
+        if (daysDifference == 1) {
+            return TOMORROW + SPACE + date.getDayOfMonth() + SPACE + date.getMonth();
+        }
```

Comment:
Don't think we should be including the day + month in this method, should be extracted to another method.